GTG/plugins/send_email/sendEmail.py

```diff
@@ -55,18 +74,58 @@ def deactivate(self, plugin_api):
         Desactivates the plugin.
         """
         # everything should be removed, in case a task is currently opened
-        try:
+
+        if self.tb_Taskbutton:
             self.plugin_api.remove_toolbar_item(self.tb_Taskbutton)
-        except:
-            pass
+            self.tb_Taskbutton = False
+            self.plugin_api.remove_toolbar_item(self.tb_Managerbutton)
+            self.tb_Managerbutton = False
+
+        if self.tb_Managerbutton:
+            self.plugin_api.remove_toolbar_item(self.tb_Managerbutton)
+            self.tb_Managerbutton = False
+
+
+
+    def selection_changed(self, selection):
+        """
+        To select multiple tasks in manager
+        deactivates button if no task selection_changed
+        """
+        if selection.count_selected_rows() > 0:
+            self.tb_Managerbutton.set_sensitive(True)
+        else:
+            self.tb_Managerbutton.set_sensitive(False)
+
+    def newButton(self, plugin_api):
+        """ Initialize all the GTK widgets """
+        self.button = True
+        self.button = Gtk.ToolButton()
+        self.button.set_sensitive(False)
+        self.button.set_icon_name('mail-send')
+        self.button.set_is_important(True)
+        self.button.set_label(_("Send via email"))
```

Comment:
Just return button instead of setting this class variable, something like

```python
button = Gtk.ToolButton()
# ... rest of code ...
return button
```